src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -397,32 +394,74 @@ public record Scaleset(
 
 ) : EntityBase();
 
+public record Container(string ContainerName)
+{
+    public string ContainerName { get; } = ContainerName.All(c => char.IsLetterOrDigit(c) || c == '-') ? ContainerName : throw new ArgumentException("Container name must have only numbers, letters or dashes");
+}
+
+public record Notification(
+    DateTime? Timestamp,
+    Container Container,
+    Guid NotificationId,
+    NotificationTemplate Config
+) : EntityBase();
 
 public record BlobRef(
     string Account,
-    Container Container,
-    string Name
+    Container container,
+    string name
 );
 
-
 public record Report(
-    string? InputURL,
+    string? InputUrl,
     BlobRef? InputBlob,
-    string? Executable,
+    string Executable,
     string CrashType,
     string CrashSite,
-    List<string> CallStack,
-    string CallStackSha256,
-    string InputSha256,
+    IEnumerable<string> CallStack,
```

Comment:
Why the change to IEnumerable? All of these fields are of known size hen they are created. so LIstor even array would make more sense. We can generalize in the code instead of the data structure.

Reply:
Good point, I switched it back